src/components/Checkout/OrderSummary.js

```diff
@@ -40,7 +38,7 @@ const OrderSummary = ({ checkout }) => {
     const { error } = await stripe.confirmPayment({
       elements,
       confirmParams: {
-        return_url: 'http://localhost:3000/checkout',
+        return_url: FRONT_END_BASE_URL,
```

Comment:
Do you want to send it to `/checkout` as you had before?

Reply:
I initially had checkout since i didnt have any other pages. Eventually i will use order page for the return url. For now instead of sending the user to the same checkout page im sending them to cart page